com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettingsRendering.cs

```diff
@@ -40,6 +40,30 @@ bool overdraw
             }
         }
 
+        DebugOverdrawMode m_OverdrawMode = DebugOverdrawMode.All;
+
+        internal DebugOverdrawMode overdrawMode
+        {
+            get => m_OverdrawMode;
+            set
+            {
+                m_OverdrawMode = value;
+                UpdateDebugSceneOverrideMode();
+            }
+        }
+
+        int m_MaxPixelCost = 10;
+
+        internal int maxPixelCost
+        {
+            get => m_MaxPixelCost;
+            set
+            {
+                m_MaxPixelCost = value;
+                UpdateDebugSceneOverrideMode();
```

Comment:
Not sure why you need this as it will only check wireframeMode which is not what you changed here.

Reply:
Agree, this seems redundant. The one above you'd need to keep, if you get rid of the separate `overdraw` boolean.